vng_api_common/utils.py

```diff
@@ -110,3 +110,19 @@ def request_object_attribute(url: str, attribute: str, resource: Union[str, None
         logger.warning("%s was retrieved from %s with the %s: %s", attribute, url, exc.__class__.__name__, exc)
         result = ''
     return result
+
+
+def generate_unique_identification(instance, date_field_name):
+    from .models import IssuedIdentifications
+    model_name = instance._meta.model_name.upper()
+    year = instance.registratiedatum.year
```

Comment:
you're not using the `date_field_name` parameter, but hardcoding here

Reply:
whoops